conf/rosalind.config

```diff
@@ -0,0 +1,28 @@
+params {
+	config_profile_description = 'Kings College London Rosalind HPC'
+	config_profile_contact = 'Theo Portlock'
+	config_profile_url = 'https://www.rosalind.kcl.ac.uk/'
+}
+
+singularity {
+	enabled = true
+	autoMounts = true
+	docker.enabled = false
+}
+
+params {
+	max_memory = 64.GB
+	max_cpus = 16
+	max_time = 24.h
+	partition = 'shared'
+}
+
+process {
+	executor = 'slurm'
+	maxRetries = 3
+	clusterOptions = { "--partition=$params.partition ${params.clusterOptions ?: ''}" }
```

Comment:
The condition there looks a bit broken, is  the `?` in the right place?

Reply:
Good spot, I have removed the broken clusterOptions and replaced with just the partition specification. I have tested this solution also and it works as expected

Reply:
The only issue  I see here is that it would maybe require all nf-core pipelines to a `$params.partition` defined in their JSON, but I'm not 100% sure (i gues sif you tested it, it should work - have you tried different nf-core pipelines). It would be very important to document this, if you can't define it dynamically based on resource requirements.

Reply:
Exactly, I have now ran multiple partitions with multiple nf-core pipelines including rnaseq without error. I also updated the README about the default partition and how to specify other partitions.